miniapps/tools/makefile

```
@@ -25,10 +25,11 @@ include $(DEFAULTS_MK)
 MFEM_LIB_FILE = mfem_is_not_built
 -include $(CONFIG_MK)
 
-
 SEQ_MINIAPPS = display-basis load-dc convert-dc get-values lor-transfer \
 	check-tmop-metric tmop-metric-magnitude
-PAR_MINIAPPS = plor-transfer
+
+PAR_MINIAPPS = nodal_transfer plor-transfer
+
```

No new lines (for consistency with other makefiles)

```suggestion
PAR_MINIAPPS = nodal_transfer plor-transfer
```

---

done